.automation/build_archi_diagram.py

```diff
@@ -0,0 +1,77 @@
+# !/usr/bin/env python3
+"""
+Generate architecture diagram of Mega-Linter
+See how to do it at https://kroki.io/#how
+"""
+import base64
+import tempfile
+import zlib
+from string import Template
+
+# pylint: disable=import-error
```

Comment:
Recommend documenting why a linter needs to be suppressed when this comes for the sake of maintainability. Otherwise we will never know whether or when we can remove a suppression.

Reply:
copy/paste from other scripts in Mega-Linter ;-)

Reply:
Fair enough, ha ha.

Reply:
with all changes I've on my todo list, I'm not at one more :)

Reply:
Yeah, I don't want to be putting the onus on contributors to fix our pre-existing issues.

Reply:
Good discovery that this Pylint suppression turned out to be unnecessary. I will try to remember to take them out where possible as I touch the files that have unnecessary suppressions.